Makefile

```diff
@@ -100,33 +90,28 @@ bin/terrafile:
 .PHONY: install-konduit
 install-konduit: bin/konduit.sh ## Install the konduit script, for accessing backend services
 
-.PHONY: terrafile
-terrafile: bin/terrafile
-	./bin/terrafile -p terraform/application/vendor/modules \
-		-f terraform/application/config/$(CONFIG)/Terrafile
-
-.PHONY: terraform-init
-terraform-init: set-resource-group-name set-storage-account-name set-azure-account terrafile
-	$(if $(DOCKER_IMAGE), , $(error Missing environment variable "DOCKER_IMAGE"))
-
-	$(eval export TF_VAR_docker_image=$(DOCKER_IMAGE))
-	$(eval export TF_VAR_config_short=$(CONFIG_SHORT))
-	$(eval export TF_VAR_service_short=$(SERVICE_SHORT))
-	$(eval export TF_VAR_azure_resource_prefix=$(AZURE_RESOURCE_PREFIX))
+terraform-init: composed-variables bin/terrafile set-azure-account ## Initialize terraform for AKS
+	$(if ${DOCKER_IMAGE_TAG}, , $(eval DOCKER_IMAGE_TAG=main))
 
+	./bin/terrafile -p terraform/application/vendor/modules -f terraform/application/config/$(CONFIG)/Terrafile
 	terraform -chdir=terraform/application init -upgrade -reconfigure \
-		-backend-config=resource_group_name=$(RESOURCE_GROUP_NAME) \
-		-backend-config=storage_account_name=$(STORAGE_ACCOUNT_NAME) \
-		-backend-config=key=$(or $(TERRAFORM_BACKEND_KEY),terraform.tfstate)
+		-backend-config=resource_group_name=${RESOURCE_GROUP_NAME} \
+		-backend-config=storage_account_name=${STORAGE_ACCOUNT_NAME} \
+		-backend-config=key=${ENVIRONMENT}_kubernetes.tfstate
```

Comment:
the backend key needs to match the existing setup, i.e. = terraform.tfstate if TERRAFORM_BACKEND_KEY doesn't exist.